rocAL/CMakeLists.txt

```diff
@@ -189,6 +190,10 @@ if(NOT HALF_FOUND)
     set(BUILD_ROCAL false)	
     message("-- ${Yellow}NOTE: rocAL library requires HALF, Not Found ${ColourReset}")	
 endif()
+if(NOT SndFile_FOUND)	
```

Comment:
rocAL can be build without audio support. Add a warning and say audio will not be supported since snddecode is not found.

Reply:
Done